docs/tasks/istio-installation.md

```diff
@@ -1,37 +1,105 @@
 ---
-title: Installing Istio
-headline: Installing Istio
+title: Istio Installation
+headline: Istio Installation
 sidenav: doc-side-tasks-nav.html
 bodyclass: docs
 layout: docs
 type: markdown
 ---
 {% capture overview %}
-This task shows how to install Istio in a Kubernetes cluster. You'll learn
+This page shows how to get started using Istio in a Kubernetes cluster. You'll learn
 how to install Istio and get it initially configured and running.
 {% endcapture %}
 
 {% capture prerequisites %}
-* Do this.
-* Do this too.
+* The following instructions assume you have access to a Kubernetes cluster. To install Kubernetes locally, try [minikube](https://github.com/kubernetes/minikube).
+* If you are using GKE, please make sure you are using static client certificates before fetching cluster credentials:
+```bash
+gcloud config set container/use_client_certificate True
+gcloud container clusters get-credentials cluster-2 --zone us-central1-a --project istio-demo
+```
+* Ensure curl is present, or clone the github [istio](https://github.com/istio/istio) repository to access the files locally.
 {% endcapture %}
 
 {% capture steps %}
-## Doing ...
+## Install Istio on an existing Kubernetes cluster
+For alpha release, Istio must be installed in the same Kubernetes namespace as the applications. Instructions below will deploy Istio in the "default" namespace. They can be modified for deployment in a separate namespace.
 
-1. Do this.
-1. Do this next. Possibly read this [related explanation](...).
+1. Source Istio configuration file:
+```bash
+source <(curl https://raw.githubusercontent.com/istio/istio/change_install/istio.VERSION)
+```
+2. Install the Istio core components (Istio-Manager, Mixer, Ingress-Controller, and Istio-CA):
+```bash
+kubectl apply -f https://raw.githubusercontent.com/istio/istio/master/kubernetes/istio.yaml
+```
+3. Download one of the [istioctl](../reference/istioctl.md) client binaries corresponding to your OS: `istioctl-osx`, `istioctl-win.exe`,
```

Comment:
Where is the link to the binary?
Its in some google store bucket right? This is what I meant by github release. When we release Alpha, github will allow us to host all the release artifacts. We can then point people to the appropriate istio binaries.. Heck, we can even point people to a shell script that detects OS and downloads the appropriate binary :)